ai-decision-services/ai-ad-dpp-engine/prepare-config-data-and-app/prepare-config-data-and-app.md

```diff
@@ -0,0 +1,217 @@
+    },
+    "serviceApiConfiguration": {
+        "anomalyDetection": {
+            "serviceEndpoint": null,
+            "profileName": "DEFAULT",
+            "projectId": "<ad-project-ocid>",
+            "compartmentId": "<your-compartment-ocid>"
+        }
+    }
+}
+```
+
+**Inferencing config** 
+
+```java
+{
+    "inputSources": [
+        {
+            "dataframeName": "S1",
+            "type": "object-storage",
+            "namespace":"<your-namespace>",
+            "bucket":"inferencing-data-bucket",
+            "objectName": "TestData.csv"
+        }
+    ],
+    "phaseInfo": {
+        "connector": {
+            "namespace":"<your-namespace>",
+            "bucket":"output-bucket",
+            "objectName": "metadata.json"
+        }
+    },
+    "processingSteps": [
+        {
+            "stepType": "singleDataFrameProcessing",
+            "configurations": {
+                "dataframeName": "S1",
+                "steps": [
+                    {
+                        "stepName": "format_timestamp",
+                        "args": {}
+                    }
+                ]
+            }
+        }
+    ],
+    "stagingDestination": {
+        "combinedResult":"S1",
+        "type": "object-storage",
+        "namespace":"<your-namespace>",
+        "bucket": "staging-bucket",
+        "folder": "training_processed_data"
+    },
+    "outputDestination": {
+        "type": "object-storage",
+        "namespace":"<your-namespace>",
+        "bucket": "output-bucket",
+        "objectName": "model_info.json",
+    },
+    "serviceApiConfiguration": {
+        "anomalyDetection": {
+            "serviceEndpoint": null,
+            "profileName": "DEFAULT",
+            "projectId": "<ad-project-id>",
+            "compartmentId": "<your-compartment-ocid>",
+            "modelId": "<>"
+        }
+    }
+}
+```
+
+  
+
+## Driver code
+
+This is an executable provided by us. Here is our latest driver code that can conduct data preprocessing, training and inferencing: [link](https://github.com/bug-catcher/oci-data-science-ai-samples/blob/master/ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/example_code/df_driver.py).
+
+## Training/Inferencing datasets
+
+ Here we attached a usable set of training and testing data in csv and parquet. 
+
+[TestData.csv](./files/TestData.csv) [TrainData.csv](./files/TrainData.csv) [TrainData.parquet](./files/TrainData.parquet)
+
+## Library
+
+Library is an archive.zip incorporating transformers we offer and user defined functions. Here is the link of an introduction to our [offered transformers](../optional/Introduction-to-Transformers-for-Data-Preprocessing.md) in this bucket. And here is a [doc](https://github.com/bug-catcher/oci-data-science-ai-samples/blob/master/ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/prepackaged_dataflow_applications.md) describing how to generate an archive.zip.
```

Comment:
nit: Library is an *archive*, not "archive.zip"

archive.zip is a name we have given to the file